cadence/contracts/bridge/FlowEVMBridge.cdc

```diff
@@ -978,6 +923,68 @@ contract FlowEVMBridge : IFlowEVMNFTBridge, IFlowEVMTokenBridge {
         FlowEVMBridgeUtils.mustSafeTransferERC721(erc721Address: bridgedAssociation, to: to, id: evmID)
     }
 
+    /// Handle permissionlessly onboarded NFTs where the bridge deployed and manages the non-native contract
```

Comment:
No logical changes made to this method, just moved it down to `access(self)` handlers for consistency.